### DISCUSSION TITLE
Resize with flexbox

### DISCUSSION THREAD

**Discussion by jzbuchalski** [TanStack/table#2425] [General]
Flexbox experts... easy one for you I hope.  In Tanner's resize example, if I resize the "Last Name" column at the end, making the column larger, it resizes the previous column "First Name" smaller.  (1) What flexbox property causes the previous column to shrink? and (2) is that preventable?  Thanks in advance!

[https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/full-width-resizable-table](url)

**Comment by Jucesr** [comment]
Just now I decided to upgrade from V6 to V7. I got everything right but the resize functionality. In V6 when resizing It does not affect other columns but in V7 it does.

React V6: https://giphy.com/gifs/Jpp6HWUT1GR2IJMCSv/fullscreen
React V7; https://giphy.com/gifs/dCR44Nywbz7wDNjwVi/fullscreen

My Code: https://codesandbox.io/s/cool-fermi-6jtgo?file=/src/Table.js

What am I missing?

**Reply by ddcech** [reply]
I have the same issue. I may need to rewrite my project to not use flexbox to get the standard table behaviour.

**Reply by wwsskkaa** [reply]
@ddcech hi! im having the same issue, im really new to this but I really need to get the resizing to work with the standard table behavior in v7, by not looking flexbox, do u use any of their layout plugin hooks? is there anything I need to watch out for? I am wondering if there is a codesandbox of your working code for me to take a look at?
